lib/modules/datasource/aws-rds/readme.md

```diff
@@ -77,3 +81,30 @@ spec:
   # amiFilter=[{"Name":"engine","Values":["mysql"]},{"Name":"engine-version","Values":["5.7"]}]
   engineVersion: 5.7.34
 ```
+
+#### Using Terraform, `aws-rds` datasource and Aurora MySQL
+
+Here is the Renovate configuration to use Terraform, `aws-rds` and Aurora MySQL:
+
+```json
+{
+  "customManagers": [
+    {
+      "description": "Update RDS",
+      "customType": "regex",
+      "fileMatch": [".+\\.tf$"],
+      "matchStrings": [
+        "\\s*#\\s*renovate:\\s*amiFilter=(?<lookupName>.+?) depName=(?<depName>.*) versioning=(?<versioning>.*)\\s*.*_version\\s*=\\s*\"(?<currentValue>.*)\""
```

Comment:
issue: Just saw this by accident. `amiFilter` seems to be wrong here and a copy/paste error from the very beginning as we are updating RDS versions and not an AMI id.

Reply:
Can you suggest the correct code? I don't understand this enough to fix it.

And thank you for the review comment! :heart:

Reply:
As `amiFilter` is not used in the code, I suggest to replace `amiFilter` by `rdsFilter` in the readme file.